Dockerfile

```diff
@@ -13,12 +12,19 @@ RUN apt-get install -y nodejs
 # Install wget used by crash reporter
 RUN apt-get install -y wget
 
-# Install python-dbusmock
-RUN apt-get install -y python-dbusmock
+# Install python-dbus
+RUN apt-get install -y python-dbus
+RUN pip install python-dbusmock
```

Comment:
It looks like it can make sense to port this piece to all other release branches.
Just to avoid issues with `python-dbusmock` in case there will be a need to rebuild a Docker container for any of them.

Reply:
@alexeykuzmin the newer branches use a newer base docker image which already pulls in a newer version of python-dbusmock, so it is only really needed for 3-1-x

Reply:
@jkleinsc 

I've recently built a Docker container from the `4-2-x` branch `Dockerfile` to run tests in, and often see exactly the same failures because of dbus timeouts.

So it looks like the current Docker containers used for E4 builds just happen to have a version of `python-dbusmock` not affected by that issue.
So I _think_ that `pip install python-dbusmock` should be used for E4+ builds as well to make sure we always use a newer version of `python-dbusmock`, which doesn't have the timeout issue.

Reply:
By the way, do you know exactly which version of `python-dbusmock` works fine?
Should be specify this version in the `Dockerfile` explicitly?